lib/Dialect/TritonGPU/Transforms/AccelerateMatmul.cpp

```diff
@@ -28,13 +28,14 @@ namespace {
 static int getMMAVersionSafe(int computeCapability, DotOp op) {
   // List supported mma version in order of preference.
   SmallVector<int> versionsSupported;
+  op.emitRemark() << "Your computeCapability is " << computeCapability;
   if (computeCapability < 75) {
     versionsSupported = {1};
   } else if (computeCapability < 90) {
     versionsSupported = {2};
   } else if (computeCapability < 100) {
     versionsSupported = {3, 2};
-  } else if (computeCapability < 110) {
+  } else if (computeCapability < 130) {
     versionsSupported = {5, 2};
```

Comment:
Consumer cards only support Ampere-style mma. So we only the version 2 is supported.

Reply:
If you still get an error go to AccelerateMatmul.cpp and just delete the 5 and leave it like:
```
  } else if (computeCapability < 130) {
    versionsSupported = {2};
```

As @masahi pointed, consumer RTX 5090 only supports version 2.

Reply:
Deleting the 110 branch is wrong. Need to add a 130 branch with only version {2}.